### DISCUSSION TITLE
Bug: Overscroll animation in android 12

### DISCUSSION THREAD

**Discussion by gartzen** [TwidereProject/TwidereX-Android#508] [General]
Hi,

I've been happilty using twidereX a while in LineageOS 18.1. This last week I upgraded to 19.1 (android 12), and they've forced overscroll animation everywhere, which I don't like, but I'm getting used to it. The problem with TwidereX (not Twidere) is that this animation is showed horizontally in the timeline: when you scroll down, animation goes right to left and back. And not when yoy hit the top or the bottom of the list, but every time you tap and scroll! 

I hope my report helps a bit, and thanks for making such a great app!

**Comment by Tlaster** [comment]
It should fix after version 1.7.0-beta01

**Reply by slopsommelier** [reply]
When is that expected

**Reply by gartzen** [reply]
https://github.com/TwidereProject/TwidereX-Android/releases

it's there, under assets.
